backend/bom-bom-server/src/main/java/me/bombom/api/v1/member/domain/Member.java

```diff
@@ -21,12 +22,24 @@
 @Entity
 @Getter
 @NoArgsConstructor(access = AccessLevel.PROTECTED)
+@Table(
+        name = "member",
+        uniqueConstraints = {
+                @jakarta.persistence.UniqueConstraint(columnNames = {"provider", "providerId"})
+        }
+)
```

Comment:
p2 

이렇게 하면 훨씬 더 가독성이 좋을 것 같아요 😎

```suggestion
@Table(
        name = "member",
        uniqueConstraints = @UniqueConstraint(columnNames = {"provider", "providerId"})
)
```